apps/codesandbox-react-next-template/src/index.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { DefaultCustomizations } from '@uifabric/theme-samples';
+import {
+  initializeIcons,
+  Checkbox,
+  Dropdown,
+  DropdownMenuItemType,
+  SearchBox,
+  Slider,
+  SpinButton,
+  Stack,
+  TextField,
+  ThemeProvider,
+  Toggle,
+} from '@fluentui/react-next';
```

Comment:
should this be using react-northstar or should the PR be renamed to Codesandbox react-next template?

Reply:
Both templates are included, i adjusted the title 😃